src/components/track-tile/TrackTileSkeleton.tsx

```diff
@@ -29,27 +26,25 @@ const styles = StyleSheet.create({
 export const TrackTileSkeleton = () => {
   const trackTileStyles = useThemedStyles(createStyles)
   return (
-    <TrackTileContainer>
-      <View style={styles.mainContent}>
-        <View style={styles.metadata}>
-          <View style={[trackTileStyles.imageContainer, trackTileStyles.image]}>
-            <ImageSkeleton styles={{ root: trackTileStyles.image }} />
-          </View>
+    <TrackTileRoot>
+      <View style={styles.metadata}>
+        <View style={[trackTileStyles.imageContainer, trackTileStyles.image]}>
+          <ImageSkeleton styles={{ root: trackTileStyles.image }} />
+        </View>
 
-          <View style={[trackTileStyles.titles]}>
-            <View style={trackTileStyles.title}>
-              <Skeleton style={styles.skeleton} width='80%' height='80%' />
-            </View>
-            <View style={[trackTileStyles.artist, { width: '100%' }]}>
-              <Skeleton style={styles.skeleton} width='60%' height='80%' />
-            </View>
+        <View style={[trackTileStyles.titles]}>
+          <View style={trackTileStyles.title}>
+            <Skeleton style={styles.skeleton} width='80%' height='80%' />
```

Comment:
I think it would improve the 'breakability' of the code if we could pass the width and height as numbers

Reply:
yeah totally agree! i can look into this when i re-use the track tile for profile page. at least for this pr, it looks changed when really it was just a reformat cause the outer view changed